docs/get-started.md

```diff
@@ -95,6 +99,9 @@ Once you have installed [Rider](https://www.jetbrains.com/rider/), install the [
 
 <%(/Expandable)%>
 
+<%(Callout type="alert" message="You need select Linux Build Support as in your setup because all server-workers in a cloud deployment run in a Linux environment. When you come to build your project, in the `Assets/Fps/Config/BuildConfiguration`, do not change the `UnityGameLogic Cloud Environment` from Linux.")%>
```

Comment:
nice red warning :D 

though not sure the sentence makes complete sense?

"You need to select Linux Build Support when installing Unity, because all server-workers are started inside a Linux environment when running a cloud deployment. When you build your project, do not change the "UnityGameLogic Cloud Environment" from Linux to another build target in the build configuration located inside `Assets/Fps/Config/BuildConfiguration` of your Unity project."

What do you think?

Reply:
HA HA no it's copy-paste garbage.